src/sample/loginController.java

```diff
@@ -39,23 +43,81 @@ public void openBrowser() throws URISyntaxException, IOException {
     openLink();
   }
 
-  public void SignIn(MouseEvent event) throws IOException {
-    if (rbStudent.isSelected()) {
-      switchTo(event, "stuDash.fxml");
-    } else if (rbLecturer.isSelected()) {
-      switchTo(event, "lecDash.fxml");
+
+
+  public void SignIn(MouseEvent event) throws IOException{
+    if (matricNo.getText().isBlank()==true && textPassword.getText().isBlank()==true)
```

Comment:
you should use `password` instead of `textPassword`, `textPassword` is just a visual thing to allow user to see, but it should not serve as the main method to get password from users